server/itx/server.go

```diff
@@ -231,9 +232,18 @@ func StartServer(ctx context.Context, svr *Server, probeSvr *probe.Server, cfg c
 			log.L().Panic("Failed to stop server.", zap.Error(err))
 		}
 	}()
+	if _, isGateway := cfg.Plugins[config.GatewayPlugin]; isGateway && cfg.API.ReadyDuration > 0 {
+		// wait for a while to make sure the server is ready
+		// The original intention was to ensure that all transactions that were not received during the restart were included in block, thereby avoiding inconsistencies in the state of the API node.
+		log.L().Info("Waiting for server to be ready.", zap.Duration("duration", cfg.API.ReadyDuration))
+		readyTimer := time.NewTimer(cfg.API.ReadyDuration)
+		<-readyTimer.C
+		readyTimer.Stop()
```

Comment:
can simplify to `time.Sleep(cfg.API.ReadyDuration)`?

Reply:
fixed